### PR TITLE
Clarify the documentation about how to load up the data for testing purposes

### PR DESCRIPTION
**Pull request by dannymandel**
Hi @hyunssong I went over my notes and I think this should hopefully clarify things.  I've placed a dump of iSamples Central in `/var/local/data/` on mars.cyverse.org.  You can use this to load up your local database for dumping purposes.

Please try this workflow out and add comments about things that are confusing.

### CODE REVIEW

**Comment by hyunssong** [comment]
Thanks @dannymandel!

File: README.md [hunk 1]
```diff
@@ -64,6 +64,17 @@ brew services start postgres
...
+
+The first thing you'll want to do is dump the postgres database (or grab it from `/var/local/data/` on mars.cyverse.org).  The following command will dump an existing iSC postgres database and create a `.sql` file that you will load up in your local database:
+
+`pg_dump -a -U isb_writer -h localhost -d isb_1 > isamples_data_only.sql` -- note that the `-a` switch specifies data-only.  Given the history of the project and the rate at which things change, it is safer to *not* include the table creation statements in the dumped .sql file.
+
+Then, you'll want to transfer that dumped file to your local machine.  It's likely faster to gzip it first on the server and gunzip it locally.  Once you've transferred to your local machine, bringing up a local Docker iSB container will create the necessary tables and indexes so that the load may proceed.  After you've done that (and created the necessary tables and indexes), you may load up the postgres dump like so:
```
Comment (hyunssong) [inline README.md:74]: When you mention a local docker isB container, would this mean creating a new docker container?

File: docs/indexing_integration_test.md [hunk 2]
```diff
@@ -13,8 +13,10 @@ There's a script in `scripts/indexing_integration_test/manage_things_with_ids.py
 ```
 python manage_things_with_id.py -d "db_url" dump --input_file /tmp/source_ids --output_file /tmp/dumped_things
```
Comment (hyunssong) [inline docs/indexing_integration_test.md:14]: I am assuming that this db_url here would be the database in the "local docker isB container", is this right?

File: README.md (hunk 1 above)
Comment (dannymandel) [reply README.md:74]: Yep, simply bringing up the container will make a new one.

File: docs/indexing_integration_test.md (hunk 2 above)
Comment (dannymandel) [reply docs/indexing_integration_test.md:14]: yeah, though I've actually been using the config option when I've been testing:

e.g.

```
python scripts/indexing_integration_test/manage_things_with_ids.py --config ./isb.cfg load --input_file /tmp/dumped_things.csv
```

**Pull request merged**